src/connected/binding.ts

```diff
@@ -107,6 +108,24 @@ export class BindingService {
     return bindingsPerConnectionId;
   }
 
+  async assistBinding(params: AssistBindingParams) {
```

Comment:
This will be called when user has attempted to open hotspot in IDE, there is a connection configured but no binding.